### PR TITLE
feature: add new language (lt-LT)

### PR DESCRIPTION
**Pull request by AndreyNegrov**
add new language (lt-LT)

### CODE REVIEW

File: locales/lt-LT/messages.po [hunk 1]
```diff
@@ -0,0 +1,1833 @@
...
+
+msgid "OK"
+msgstr "Gerai"
+
+msgid "one association pair"
+msgstr "vienas ryšys per porą"
+
+msgid "One or more QTI components are not supported by the system. The system returned the following error: %s\n"
+""
+msgstr "Vienas ar daugiau QTI komponentų nėra palaikomas sistemos. Sistema gražino šią klaidą: %s\n"
```
Comment (tikhanovichA) [inline locales/lt-LT/messages.po]: is it LT version?

File: locales/lt-LT/messages.po [hunk 2]
```diff
@@ -0,0 +1,1833 @@
...
+msgstr "Prašome nurodyti teisingą atsakymą ir taškus žemiau."
+
+msgid "Please define the correct response below."
+msgstr "Prašome nurodyti teisingą atsakymą žemiau."
+
+msgid "Please define the correct response using the slider."
+msgstr "Prašome nurodyti teisingą atsakymą naudojant slankiklį."
+
+msgid "Please enter the score for the given hottexts."
+msgstr ""
```
Comment (tikhanovichA) [inline locales/lt-LT/messages.po]: Is it should be not translatable?

File: locales/lt-LT/messages.po [hunk 3]
```diff
@@ -0,0 +1,1833 @@
...
+msgstr "Prašome pasirinkti teisingus karšto taško pasirinkimus apačioje."
+
+msgid "Please select the correct hottext choices below."
+msgstr "Prašome pasirinkti teisingus karšto teksto pasirinkimus apačioje."
+
+msgid "Please set the correct associations by linking the choices."
+msgstr "Prašome nustatyti teisingus ryšius sujungdami pasirinkimus."
+
+msgid "Please the score of each hotspot choice."
+msgstr ""
```
Comment (tikhanovichA) [inline locales/lt-LT/messages.po]: Is it should be not translatable?

File: locales/lt-LT/messages.po [hunk 4]
```diff
@@ -0,0 +1,1833 @@
...
+msgstr "Portatyvus elemento modelis %s nerastas. Reikalaujamas plėtinys galbūt nerastas"
+
+msgid "Position one or more points on a picture (response areas are not displayed)."
+msgstr "Paženklinti vieną arba daugiau taškų paveikslėlyje (pasirinkimų laukai nėra atvaizduoti)."
+
+msgid "positive"
+msgstr "pozityvus"
+
+msgid "Power form"
+msgstr ""
```
Comment (tikhanovichA) [inline locales/lt-LT/messages.po]: Is it should be not translatable?

File: locales/lt-LT/messages.po [hunk 5]
```diff
@@ -0,0 +1,1833 @@
...
+
+msgid "Provides a hint to the candidate as to the expected number of lines of input required. A line is expected to have about 72 characters."
+msgstr "Pateikia kandidatui užuominą, koks yra norimas kiekis eilučių, privalomam teksto įvedimui. Eilutė laikoma ta, kuri turi apie 72 simbolius."
+
+msgid "Provides a hint to the candidate as to the expected overall length of the desired response measured in number of characters."
+msgstr ""
+
+msgid "Provides a hint to the candidate as to the expected overall length of the desired response measured in number of characters. This is not a validity constraint."
+msgstr ""
```
Comment (tikhanovichA) [inline locales/lt-LT/messages.po]: Is it should be not translatable?

**Comment by tikhanovichA** [comment]
I see a lot of not translatable line. Is it ok?

File: locales/lt-LT/messages.po (hunk 2 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: yes, it's normal

File: locales/lt-LT/messages.po (hunk 3 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: it's normal

File: locales/lt-LT/messages.po (hunk 4 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: it's normal

File: locales/lt-LT/messages.po (hunk 5 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: it's normal

File: locales/lt-LT/messages.po [hunk 6]
```diff
@@ -0,0 +1,1833 @@
...
+msgid "An unexpected error occurred during the import of the IMS QTI Item Package. "
+msgstr "Netikėta klaida įvyko IMS QTI Elemento Paketo importavimo metu."
+
+msgid "An unexpected error occurred during the import of the IMS QTI Item Package. The system returned the following error: \"%s"
+msgstr "Netikėta klaida įvyko IMS QTI Elemento Paketo importavimo metu. Sistema gražino šią klaidą: \"%s"
+
+msgid "An unexpected error occurred during the import of the QTI Item. The system returned the following error: %s\n"
+""
+msgstr "Netikėta klaida įvyko QTI Elemento importavimo metu. Sistema gražino šią klaidą: \"%s\n"
+" "
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]: Why is there a space?

File: locales/lt-LT/messages.po [hunk 7]
```diff
@@ -0,0 +1,1833 @@
...
+msgid "Lower Bound"
+msgstr "Apatinė Riba"
+
+msgid "Lower than or equal"
+msgstr "Mažiau negu arba lygu"
+
+msgid "Malformed XML[%s]:\n"
+"%s"
+msgstr "Klaidingas XML[%s]:\n"
+"%1"
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]: Why it's `%1` instead `%s`

File: locales/lt-LT/messages.po [hunk 8]
```diff
@@ -0,0 +1,1833 @@
...
+msgstr "Pi"
+
+msgid "Plain text"
+msgstr "Paprastas tekstas"
+
+msgid "Please add one or more gaps to the text in question mode."
+msgstr ""
+
+msgid "Please create areas that correspond to the response and associate them a score.\n"
+""
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]: Is this ok? It looks like it's missing smthing.

File: locales/lt-LT/messages.po [hunk 9]
```diff
@@ -0,0 +1,1833 @@
...
+msgstr "Kelio argumentas ekstraktoriui privalo būti ne tuščias masyvas."
+
+msgid "The principle identifier of the item. This identifier must have a corresponding entry in the item's metadata."
+msgstr "Pagrindinis elemento identifikatorius. Šis identifikatorius privalo turėti papildomą įrašą elemento meta duomenyse."
+
+msgid "The purpose for which the rubric is intended to be used. As a reading passage for multiple items, or instructions for a particular section, for example."
+msgstr "Tikslas dėl kurio rubriką turi būti naudojama. Pavyzdžiui, kaip skaitymui skirta ištrauka keliems elementams arba instrukcijos pasirinktai sekcijai."
+
+msgid "The QTI class \"%1$s\" located at line \"%2$d\" is currently not supported."
+msgstr "QTI klasė \"%1$s\" esanti \"%1$s\" eilutėje yra nepalaikoma."
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]: second argument should be a digit?

File: locales/lt-LT/messages.po [hunk 10]
```diff
@@ -38,21 +39,29 @@ msgid "-- Any kind of file --"
 msgstr "-- Betkoks failas --"
 
 msgid "1 item imported from the given QTI/APIP XML Item Document."
-msgstr "1 elementas buvo importuotas iš pateikto QTI/APIP XML Elemento Dokumentas."
+msgstr ""
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]: Is that correct?

File: locales/lt-LT/messages.po [hunk 11]
```diff
@@ -38,21 +39,29 @@ msgid "-- Any kind of file --"
...
 msgstr "Variantas privalo būti pasirinktas"
 
 msgid "A human interpretation of the variable's value."
 msgstr "Žmogiška kintamojo reikšmės interpretacija."
 
-msgid "A QTI component is not supported. The system returned the following error: %s\n"
-""
+msgid ""
+"A QTI component is not supported. The system returned the following error: %"
+"s\n"
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]: Why there are 3 lines? and is it ok if we divide variable type into 2 different strings?

File: locales/lt-LT/messages.po [hunk 12]
```diff
@@ -312,8 +363,12 @@ msgstr "CSV failas"
 msgid "Current value:"
 msgstr "Dabartinė reikšmė:"
 
-msgid "Currently conversion from MathML to LaTeX is not available. Editing MathML here will have the LaTex code discarded."
-msgstr "Šiuo metu konversija iš MathML į LaTeX nėra galima. Redaguojant MathML čia, LaTeX kodas bus atmestas."
+msgid ""
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]: Why you have so many initial additional lines?

File: locales/lt-LT/messages.po [hunk 13]
```diff
@@ -748,21 +840,21 @@ msgstr "Kalba"
 msgid "Large editor"
 msgstr "Didelis redaguoklis"
 
-msgid "Latex"
-msgstr "Latex"
-
 msgid "LaTeX"
 msgstr "LaTeX"
 
+msgid "Latex"
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]: How was that generated? If I remember correctly it should be LaTex. Can you confirm that?

File: locales/lt-LT/qtiItemRunner.rdf.po [hunk 14]
```diff
@@ -1,14 +1,18 @@
 msgid ""
 msgstr ""
-"Project-Id-Version: TAO 3.4.0-sprint112\n"
-"PO-Revision-Date: 2019-10-09T09:02:52\n"
-"Last-Translator: TAO Translation Team <translation@tao.lu>\n"
-"MIME-Version: 1.0\n"
+"Project-Id-Version: PACKAGE VERSION\n"
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/qtiItemRunner.rdf.po]: What's happen here?

File: locales/lt-LT/qtiItemRunner.rdf.po [hunk 15]
```diff
@@ -1,14 +1,18 @@
...
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-10-04 13:45+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
 "Language: lt-LT\n"
-"sourceLanguage: en-US\n"
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/qtiItemRunner.rdf.po]: Why did you remove it?

File: locales/lt-LT/taoQti.rdf.po [hunk 16]
```diff
@@ -1,14 +1,18 @@
...
-"Last-Translator: TAO Translation Team <translation@tao.lu>\n"
-"MIME-Version: 1.0\n"
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-10-04 13:45+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
 "Language: lt-LT\n"
-"sourceLanguage: en-US\n"
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/taoQti.rdf.po]: WHy you remove it?

File: locales/lt-LT/messages.po (hunk 1 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: yes

File: locales/lt-LT/messages.po (hunk 8 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: yes. it is normal

Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: i asked Anshul

File: locales/lt-LT/messages.po (hunk 9 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: image
yes

Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: i will fix it in translate

File: locales/lt-LT/messages.po (hunk 7 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: image

Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: you right, it's need %s, i'll fix it

File: locales/lt-LT/messages.po (hunk 13 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: in the source files it is just that

**Comment by bartlomiejmarszal** [comment]
> > There are a lot of empty first strings in each of translation. DO you know what is causing that?
> 
> yes, it are not in the source file. Anshul said it's normal

Maybe it can happen a few times but in your code, compared to previous translations there are much more these empty lines. I'm not sure if the should stay there as they mak translation less readable.

File: locales/lt-LT/messages.po (hunk 6 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: fixed

File: locales/lt-LT/messages.po (hunk 7 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: fixed

File: locales/lt-LT/messages.po (hunk 9 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: fixed

File: locales/lt-LT/messages.po (hunk 10 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: fixed

File: locales/lt-LT/messages.po (hunk 11 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: fixed

File: locales/lt-LT/messages.po (hunk 12 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: fixed

**Comment by AndreyNegrov** [comment]
> > > There are a lot of empty first strings in each of translation. DO you know what is causing that?
> > 
> > 
> > yes, it are not in the source file. Anshul said it's normal
> 
> Maybe it can happen a few times but in your code, compared to previous translations there are much more these empty lines. I'm not sure if the should stay there as they mak translation less readable.

I fix this. can you see? my be it bacame empty lines

**Comment by AndreyNegrov** [comment]
> I see a lot of not translatable line. Is it ok?

it's normal
